.github/actions/deploy-happy-stack/action.yml

```diff
@@ -52,23 +56,25 @@ runs:
         ENV_FILE: ${{ inputs.env-file }}
         OPERATION: ${{ inputs.operation }}
         SKIP_MIGRATIONS: ${{ inputs.skip-migrations }}
+        WORKING_DIRECTORY: ${{ inputs.working-directory }}
       shell: bash
       run: |
```

Comment:
you can set a default working directory rather than cding

Reply:
`working-directory: ${{ inputs.working-dir }}`

Reply:
ah yeah, thanks 👍